Documentation/contributing/development/introducing_new_crds.rst

```diff
@@ -0,0 +1,400 @@
+Introducing new CRDs
+====================
+
+Cilium uses a combination of code generation tools to facilitate adding
+CRDs to the Kubernetes instance it is installed on.
+
+These CRDs make themselves available in the generated Kubernetes client
+Cilium uses.
+
+Defining And Generating CRDs
+----------------------------
+
+Paths
```

Comment:
```suggestion
Paths:
```

Also I'd maybe move the paths after the next sentence, “Currently, two API versions exist...”